balancer-js/src/modules/exits/exits.module.ts

```diff
@@ -441,7 +521,14 @@ export class Exit {
       outputReferences,
     });
 
-    return call;
+    let tokenOutAmount = Relayer.isChainedReference(limits[0])
+      ? '0'
+      : limits[0];
+    const bptAmount = Relayer.isChainedReference(limits[1]) ? '0' : limits[1];
+    const bptIn = sender === this.relayer ? '0' : bptAmount;
+    tokenOutAmount = recipient === this.relayer ? '0' : tokenOutAmount;
```

Comment:
I feel like it would be interesting to have a few comments explaining these values so it's easier to understand what's going on.
If I understood correctly, if the recipient is the relayer, it means that this is an intermediary transaction and that we can ignore amount out to calculate deltas. Is that right?